pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -154,6 +154,14 @@
     dependencies = with super; [ plenary-nvim ];
   };
 
+  avante-nvim = (callPackage ./avante-nvim { }).overrideAttrs {
+    dependencies = with self; [
+      dressing-nvim
```

Comment:
can you also add nvim-treesitter?

we will merge https://github.com/yetone/avante.nvim/pull/496 soon which requires treesitter.

Reply:
let's add it when it's actually required